django-stubs/db/models/fields/related_descriptors.pyi

```diff
@@ -73,7 +73,7 @@ class ReverseOneToOneDescriptor(Generic[_From, _To]):
     def __set__(self, instance: _From, value: _To | None) -> None: ...
     def __reduce__(self) -> tuple[Callable[..., Any], tuple[type[_To], str]]: ...
 
-class ReverseManyToOneDescriptor:
+class ReverseManyToOneDescriptor(Generic[_To]):
```

Comment:
Hm, I think I'm gonna put this change in a separate PR, coming first, as I realised that there are more changes here since we should make all reverse managers for FK:s go via this `ReverseManyToOneDescriptor`.

Reply:
Separate PR is here: #2227